exasol_sagemaker_extension/autopilot_prediction.py

```diff
@@ -24,7 +25,7 @@ def run(self, ctx):
             os.environ["AWS_ACCESS_KEY_ID"] = aws_s3_conn_obj.user
             os.environ["AWS_SECRET_ACCESS_KEY"] = aws_s3_conn_obj.password
 
-            data_df = ctx.get_dataframe(num_rows='all')
+            data_df = ctx.get_dataframe(self.batch_size)
```

Comment:
You need to iterate over the data, at the moment you will only send the first 100 rows of the group to the endpoint.